src/default.nix

```diff
@@ -556,20 +556,9 @@
           (outputs.packages or {});
       };
 
-    projectOutputs =
-      l.map
-      (proj: outputsForProject proj)
-      translatedProjects;
+    projectOutputs = l.map outputsForProject translatedProjects;
 
-    mergedOutputs =
-      l.foldl'
-      (all: outputs:
-        all
-        // {
-          packages = all.packages or {} // outputs.packages;
-        })
-      {}
-      projectOutputs;
+    mergedOutputs = l.foldl' l.recursiveUpdate {} projectOutputs;
```

Comment:
recursiveUpdate on attrsets that contain derivations can be dangerous because derivations are attrsets as well and might end up being merged to a weird result.

Reply:
Oh right, I keep forgetting about that :sweat_smile: would the latest commit work then?